src/core/AutoRest/Program.cs

```diff
@@ -91,13 +91,6 @@ private static int Main(string[] args)
                             return 0;
                         }
 
-                        // comparison?
```

Comment:
Nah we postponed that again and again... shall we move? :-D

Right, how about I turn our CLI help from the [docs](https://github.com/Azure/autorest/blob/master/docs/user/cli.md) into markdowny `--help`